dust-lambda-extension/extensions/dust

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env bash
```

Comment:
does this need to be a bash script? what does it do which actually requires bash?

can this rely simply on `/bin/sh`

lambda runtimes are used in other envs such as https://www.localstack.cloud/ which might not contain `bash`

unless we absolutely we need `bash`, lets not rely on it